.travis.yml

```diff
@@ -88,7 +89,20 @@ script:
       ./build/sbt -Dspark.version=$SPARK_VERSION
                   -Dscala.version=$SCALA_VERSION
                   coverage test coverageReport"
-
+  # run python style checks
+  - docker exec -t ubuntu-test bash -c "
```

Comment:
Let's move as much of this as possible to run.py and keep the travis file short.